### PR TITLE
Adding reviewers for FortiManager modules

### PR DESCRIPTION
**Pull request by Ghilli3**
##### SUMMARY

Adding reviewers to BOTMETA.yml for FortiManager modules

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

BOTMETA.yml

##### ANSIBLE VERSION

```paste below
ansible 2.8.0.dev0 (fmgr_secprof_wanopt 55f120cbfc) last updated 2018/10/29 09:04:53 (GMT -600)
  config file = /etc/ansible/ansible.cfg
  configured module search path = [u'/home/cse/PycharmProjects/JPMC_FOS/ansible/lib/ansible/modules']
  ansible python module location = /home/cse/PycharmProjects/ftntcorecse-ansible/lib/ansible
  executable location = /home/cse/PycharmProjects/ftntcorecse-ansible/bin/ansible
  python version = 2.7.12 (default, Dec  4 2017, 14:50:18) [GCC 5.4.0 20160609]``

##### ADDITIONAL INFORMATION
<!--- Include additional information to help people understand the change here -->
<!--- A step-by-step reproduction of the problem is helpful if there is no related issue -->

<!--- Paste verbatim command output below, e.g. before and after your change -->
```paste below

```

### CODE REVIEW

File: .github/BOTMETA.yml [hunk 1]
```diff
@@ -923,3 +929,4 @@ macros:
   team_vultr: resmo Spredzy
   team_windows: dagwieers jborean93 jhawkesworth nitzmahone
   team_windows_core: nitzmahone jborean93
+  team_fortimanager: lweighall Ghilli3 p4r4n0y1ng ftntcorecse
```
Comment (gundalow) [inline .github/BOTMETA.yml]: Missing newline (that's what the red marker is showing)

**Review by gundalow — COMMENTED** [review]
Minor fix (& rebase needed) then this should be good to merge

**Comment by Ghilli3** [comment]
I will be re-opening a new PR soon

**Comment by gundalow** [comment]
http://docs.ansible.com/ansible/devel/dev_guide/developing_rebasing.html should help you do a clean rebase

**Pull request closed without merging**